.github/workflows/post-process.yml

```diff
@@ -44,9 +43,7 @@ jobs:
           - windows-latest
           - macos-14
         exclude:
-          - {version: {
-               python: "3.9" },
-            os: macos-14 }
+          - { os: macos-14 }
```

Comment:
The entire `exclude` block should be removed now. This was previously included because there were problems with python 3.9 support for ARM macs.

As you have it now, this `exclude` block will prevent any tests from running using the `macos-14` OS